### PR TITLE
8.0 mig website sale checkout skip payment

### PR DESCRIPTION
**Pull request by larmar**
This is the downgraded module with history #207

### CODE REVIEW

**Review by chienandalu — COMMENTED** [review]
Count with this PR as well: https://github.com/OCA/e-commerce/pull/223

File: website_sale_checkout_skip_payment/README.rst [hunk 1]
```diff
@@ -0,0 +1,63 @@
+.. image:: https://img.shields.io/badge/license-LGPL--3-blue.svg
+   :target: https://www.gnu.org/licenses/lgpl.html
+   :alt: License: LGPL-3
+
+==================================
+Website Sale Checkout Skip Payment
+==================================
+
+This module allows to logged users to checkout with no payment step. At the
```
Comment (hbrunn) [inline website_sale_checkout_skip_payment/README.rst:9]: ...allows logged in users to checkout without ...

**Review by hbrunn — APPROVED** [review]
just some comment about language

**Comment by hbrunn** [comment]
oh, i just see you should add tests too

**Pull request closed without merging**